mobile/src/main/res/values/strings.xml

```diff
@@ -196,31 +196,31 @@
     <string name="send_device_info_category_event_based_title">Event based</string>
     <string name="send_device_info_category_event_based_explanation">These Items are updated on corresponding events, e.g. an incoming phone call</string>
     <string name="send_device_info_category_schedule_based_title">Schedule based</string>
-    <string name="send_device_info_category_schedule_based_explanation">These Items are updated on a regular schedule\nNote: Lower values impair battery life more than higher ones and your device may decide to delay updates.</string>
+    <string name="send_device_info_category_schedule_based_explanation">These Items are updated on a regular schedule\nNote: Shorter intervals impair battery life more than higher ones and your device may decide to delay updates.</string>
```

Comment:
Another thing: `higher` -> `longer`

Maybe also reword this completely?
`The shorter the chosen interval is, the larger is the potential impact on the battery life of your device. Additionally, your device might not follow the chosen interval exactly, but delay the updates depending on its operating conditions.`

Reply:
Maybe `...but delay the updates.` or `...but delay the updates to save battery.`?

Reply:
As this is an explanation, I wouldn't omit (potential) delay reasons. Your latter suggestion is fine with me, I'm just not sure whether it's valid English wording. Maybe `...but delay the updates to reduce power consumption across apps.`? That way we'd kinda explain the additional delay is due to wakeup batching.